blocks/editable/index.js

```diff
@@ -72,6 +75,28 @@ function getFormatProperties( formatName, parents ) {
 
 const DEFAULT_FORMATS = [ 'bold', 'italic', 'strikethrough', 'link' ];
 
+/**
+ * Transforms internal block's representation into an Element.
+ *
+ * @param {Array} value Value to transform
+ * @return {WPElement} Element.
+ */
+export function valueToElement( value ) {
+	if ( ! Array.isArray( value ) ) {
+		return value;
```

Comment:
When will this condition pass?

Reply:
@gziolo I believe you added this?

Reply:
I added it to make the existing tests pass. Example usage:

```js
it( 'should warn when rendered with string value', () => {
    shallow( <Editable value="Uh oh!" /> );
    expect( console.error ).toHaveBeenCalled();
} );
```